### PR TITLE
feat: show deploy transactions transformed

### PR DESCRIPTION
**Pull request by bluecco**

### CODE REVIEW

**Comment by bluecco** [comment]
@janek26 updated code after starknetjs update

let me know especially about the changes on `wallet.ts`

File: packages/extension/src/shared/call/udcDeployCall.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Call, constants, validateAndParseAddress } from "starknet"
+
+import { isEqualAddress } from "../../ui/services/addresses"
+
+const { UDC } = constants
+
+export const isUdcDeployCall = (call: Call): call is Call => {
```
Comment (simonheys) [inline packages/extension/src/shared/call/udcDeployCall.ts:7]: Same comment as above?

**Review by simonheys — APPROVED** [review]

File: packages/extension/src/ui/features/accountActivity/TransactionDetailScreen.tsx [hunk 2]
```diff
@@ -48,6 +49,14 @@ export const TransactionDetailScreen: FC = () => {
     )
   }, [network.id, transactions, txHash])
 
+  // TODO: remove this when after backend update
+  const isUdcTransaction = useMemo(() => {
+    return (
+      get(transaction, "meta.transactions.entrypoint") === "deployContract" ||
+      get(transaction, "meta.transactions.entrypoint") === "declareContract"
```
Comment (janek26) [inline packages/extension/src/ui/features/accountActivity/TransactionDetailScreen.tsx:56]: can we avoid lodash get? 🙏 TS doesnt play well with it

**Review by janek26 — APPROVED** [review]

**Comment by janek26** [comment]
works on my machine!

**Pull request merged**